recipes/python-poppler/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "python-poppler" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python_poppler-{{ version }}.tar.gz
+  sha256: e6ca5c23ec02350bf2cef85a6bf9f1b261796436db478176f9d24fb14eeecc6a
+
+build:
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .  # [unix]
+  number: 0
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - poppler                             # [build_platform != target_platform]
+    - pkg-config                          # [build_platform != target_platform]
+    - clang                               # [win]
+    - lld                                 # [win]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - meson-python
+    - meson >=1.0.0
+    - wheel
+    - pip
+    - pkg-config
+    - poppler
+  run:
+    - python
+    - poppler
```

Comment:
This should be covered bt ru_exports and not listed here

Reply:
This would require poppler to have a `run_export`, right? I think this export is currently missing https://github.com/conda-forge/poppler-feedstock/blob/main/recipe/meta.yaml